contrib/neon/pagestore_smgr.c

```diff
@@ -84,6 +84,10 @@ static char *hexdump_page(char *page);
 
 const int	SmgrTrace = DEBUG5;
 
+/*
+ * Pseudo block number used to associate LSN with relation metadata (relation size */
+#define REL_METADATA_PSEUDO_BLOCKNO 0xFFFFFFFE
```

Comment:
It's a bit ugly to use this value because 0xFFFFFFFE is a valid block number. You will probably hit other problems before you hit that limit, but still. Could use 0xFFFFFFFF, i.e. InvalidBlockNumber instead.